src/main/java/org/opentripplanner/street/search/state/State.java

```diff
@@ -52,6 +53,12 @@ public class State implements AStarState<State, Edge, Vertex>, Cloneable {
   // we should DEFINITELY rename this variable and the associated methods.
   public double walkDistance;
 
+  // how far a sharing vehicle powered by battery has driven
+  public double drivenBatteryMeters;
```

Comment:
Since bikes and scooters are ridden, I would call this variable `travelledBatteryMeters` or `traversedBatteryMeters` (more correct).

Reply:
we chose traversedBatteryMeters as a new name, thanks